### PR TITLE
MP-6047 Product Approval Process + Merchant Context

### PR DESCRIPTION
**Pull request by ilyakubanov**
Ticket: https://spryker.atlassian.net/browse/MP-6047

### CODE REVIEW

File: docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/bdeba422-0437-4c39-a29f-9340eb153a6a.png?utm_medium=live&utm_source=custom)
+
+| MODULE     | DESCRIPTION                |
+|------------|----------------------------|
+| [MerchantProductApproval](https://github.com/spryker/merchant-product-approval) | This module provides merchant product approval functionality. |
+| [MerchantProductApprovalDataImport](https://github.com/spryker/merchant-product-approval-data-import) | This module provides functionality to import the merchant product approval process related data. |
+
+## Domain model
+
+The following schema illustrates the *Marketplace Product Approval Process* domain model:
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md:24]: fixed

File: docs/scos/dev/feature-walkthroughs/202200.0/product-approval-process-feature-walkthrough.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+|------------|----------------------------|
+| [ProductApproval](https://github.com/spryker/product-approval) | This module provides product approval functionality. |
+| [ProductApprovalDataImport](https://github.com/spryker/product-approval-data-import) | Module for importing Product approval statuses from .csv file. |
+| [ProductApprovalGui](https://github.com/spryker/product-approval-gui) | This module provides Zed UI interface for product approval management. |
+
+## Domain model
+
+The following schema illustrates the *Product Approval Process* domain model:
+
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/9307bb4e-6cb5-43d7-b7a5-e0b3d98ef664.png?utm_medium=live&utm_source=custom)
```
[the remark this answers is not in the document]
Comment (ilyakubanov) [reply docs/scos/dev/feature-walkthroughs/202200.0/product-approval-process-feature-walkthrough.md:29]: discussed

**Review by asmarovydlo — DISMISSED** [review]

File: docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Marketplace Product Approval Process feature walkthrough
+description: Marketplace Product Approval Process feature adds Merchant context to product approval process.
+template: feature-walkthrough-template
+---
+
+The *Marketplace Product Approval Process* feature adds a Merchant context to Product Approval Process by providing
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md:7]:
```suggestion
The *Marketplace Product Approval Process* feature adds a merchant context to Product Approval Process by providing
```

File: docs/marketplace/dev/data-import/202200.0/file-details-merchant-product-approval-status-default.csv.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: "File details: merchant_product_approval_status_default.csv"
+description: This document describes the merchant_product_approval_status_default.csv file to configure default merchant products approval statuses in your Spryker shop.
+template: import-file-template
+---
+
+This document describes the `merchant_product_approval_status_default.csv` file to configure default merchant products approval statuses in your Spryker shop.
+
+To import the file, run:
```
Comment (VadymSachenko) [inline docs/marketplace/dev/data-import/202200.0/file-details-merchant-product-approval-status-default.csv.md:9]:
```suggestion
To import the file, run the following:
```

File: docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+title: Marketplace Product Approval Process feature walkthrough
+description: Marketplace Product Approval Process feature adds Merchant context to product approval process.
+template: feature-walkthrough-template
+---
+
+The *Marketplace Product Approval Process* feature adds a merchant context to Product Approval Process by providing
+an optional `spy_merchant.default_product_abstract_approval_status` DB column where default product approval statuses for merchants are stored and can be used as default product approval statuses for merchant products. Also, the feature provides Data importer for merchant's default product approval statuses.
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-product-approval-process-feature-walkthrough.md:8]:
```suggestion
an optional `spy_merchant.default_product_abstract_approval_status` DB column where default product approval statuses for merchants are stored and can be used as default product approval statuses for merchant products. Also, the feature provides the data importer for the merchant's default product approval statuses.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**